lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -542,9 +542,7 @@ void addBranchOps(handshake::FuncOp f, ConversionPatternRewriter &rewriter) {
 
         if (auto condBranchOp = dyn_cast<mlir::CondBranchOp>(termOp))
           newOp = rewriter.create<handshake::ConditionalBranchOp>(
-              termOp->getLoc(),
-              condBranchOp.getCondition(),
-              val);
+              termOp->getLoc(), condBranchOp.getCondition(), val);
```

Comment:
This looks like a clang format patch, it would be great to split this off to its own patch.  clang-format-only patches should always be preapproved for CIRCT IMO.